cpp/src/utilities/cudf_utils.h

```diff
@@ -36,9 +36,58 @@ bool gdf_is_valid(const gdf_valid_type *valid, gdf_index_type pos) {
 		return true;
 }
 
+/**
+ * Calculates the index of the last `gdf_valid_type` element in the validity
+ * bitmask for a given column's size.
+ *
+ * @note Note that this function assumes that `gdf_valid_type` is unsigned char
+ * @note This function is different gdf_get_num_bytes_for_valids_allocation
+ * because it refers to the last `gdf_valid_type` element that refers to
+ * elements in the column, NOT the last element in the allocation
+ *
+ * @param[in] column_size the number of elements
+ * @return The index of the last `gdf_valid_type` element that refers to
+ * elements in a column of size @p column_size
+ */
+CUDA_HOST_DEVICE_CALLABLE
+gdf_size_type gdf_last_bitmask_index(gdf_size_type column_size) {
```

Comment:
Is this function never called? It's a new function in this PR, but this is the only place it appears.

Reply:
See unchecked item in PR description: 
>  Update all uses of gdf_get_num_chars_bitmask to use either gdf_last_bitmask_index or gdf_get_num_bytes_for_valids_allocation, as appropriate